packages/plugins/rudder_plugin_ios/ios/Classes/RudderSdkFlutterPlugin.m

```diff
@@ -185,15 +185,16 @@ - (RSConfig*)getRudderConfigObject:(NSDictionary*)configDict {
       [configBuilder withDataResidencyServer:EU];
   }
 
-  if([configDict objectForKey:@"dbEncryption"]) {
-    NSDictionary* dbEncryptionDict = [configDict objectForKey:@"dbEncryption"];
-    BOOL enabled = [[dbEncryptionDict objectForKey:@"enabled"] boolValue];
-    NSString* encryptionKey = [dbEncryptionDict objectForKey:@"key"];
+  // to be implemented in future
```

Comment:
add TODO: as comment prefix, IDEs hightlight it and its easier to find all your todos

Reply:
added.